src/firefly/js/charts/ui/PlotlyWrapper.jsx

```diff
@@ -280,9 +280,10 @@ export class PlotlyWrapper extends Component {
                         this.restyle(this.div, Plotly, data, dataUpdate, dataUpdateTraces);
                         break;
                     case RenderType.RELAYOUT:
-                        Plotly.relayout(this.div, layoutUpdate);
+                        Plotly.relayout(this.div, layout);
```

Comment:
This should be fine.  `layoutUpdate` may no longer be necessary since plotly may be doing it internally.